README.md

```diff
@@ -1,8 +1,6 @@
 # Contour
 Contour is a typesafe, Kotlin-first API for complex layouts on Android.
 
-This project is currently **experimental and the API subject to breaking changes without notice**. 
```

Comment:
will the API no longer break without notice?  Might be worth keeping that part til a 1.0

Reply:
I think we're at the point were we won't break anything without notice. We can still deprecate things and schedule their removal.